tests/integration/azure/test_locations.py

```diff
@@ -20,7 +27,7 @@ def save_delete_location(ws, name):
 
 
 @pytest.mark.skip
-def test_run(caplog, ws, sql_backend, inventory_schema):
+def test_run(caplog, ws, sql_backend, inventory_schema, runtime_ctx):
```

Comment:
```suggestion
def test_run(caplog, ws, sql_backend, inventory_schema, azure_cli_ctx):
```
Rewrite to use the correct test context